tsconfig.json

```diff
@@ -14,7 +14,7 @@
     "module": "esnext",
     "moduleResolution": "node",
     "noFallthroughCasesInSwitch": true,
-    "noImplicitReturns": false,
+    "noImplicitReturns": true,
```

Comment:
Is there a reason why for update this configuration

Reply:
yes. before that, I changed from `true to false`. At that time, I return the loop without ending. So that I needed to set this as false. 

Now I changed to use find function. So that I changed it back from `false to true`